shims/common/src/main/scala/org/apache/gluten/GlutenConfig.scala

```diff
@@ -1940,6 +1943,14 @@ object GlutenConfig {
       .booleanConf
       .createWithDefault(true)
 
+  val VELOX_OBJECTHASHAGG_COLLECT_REWRITE_ENABLED =
+    buildConf("spark.gluten.sql.columnar.backend.velox.objectHashAggregate.collect.rewrite.enabled")
```

Comment:
umm...the config name seems too long?

Reply:
It is indeed a bit long, but it seems difficult to shorten. How do you think it should be changed?

Reply:
how about spark.gluten.sql.columnar.backend.velox.collectRewriteEnabled?

Reply:
> how about spark.gluten.sql.columnar.backend.velox.collectRewriteEnabled?

This is not global, it just disables objectHashAggregate's collect, although currently there is only objectHashAggregate, there will be ambiguity in naming this way. cc @zhztheplayer